src/compiler/crystal/codegen/target.cr

```diff
@@ -82,8 +88,12 @@ class Crystal::Codegen::Target
     @environment.starts_with?("linux")
   end
 
+  def bsd?
+    freebsd? || dragonfly? || openbsd?
```

Comment:
```suggestion
    freebsd? || openbsd? || dragonfly?
```

Reply:
???

Reply:
I always group freebsd? and dragonfly? closer together as they are more similar. but I am fine with changing the order towards FCFS :)